tensorflow-mnist-input-data/Setup.hs

```diff
@@ -33,6 +33,9 @@ import System.FilePath ((</>))
 import System.Directory (doesFileExist)
 import qualified Crypto.Hash as Hash
 import qualified Data.ByteString.Lazy as B
+import qualified Data.ByteString.Char8 as C
```

Comment:
Yes, I've changed it to keep the representation Data.Bytestring.Lazy similar to the original implementation.